pages/common/hipstopgm.md

```diff
@@ -0,0 +1,17 @@
+# hipstopgm
+
+> Reads a HIPS file as input and returns a PGM image as output.
+> If the HIPS file contains more than one frame in sequence, hipstopgm will concatenate all the frames vertically.
```

Comment:
```suggestion
> Read a HIPS file as input and return a PGM image as output.
> If the HIPS file contains more than one frame in sequence, `hipstopgm` will concatenate all the frames vertically.
```
Note: As highlighted in the style guide we prefer:

- To have descriptions in an imperative mood (i.e "Write" instead of "Writes").
- To enclose commands inside backticks (`).